packages/flex-plugin-scripts/src/scripts/__tests__/pre-script-check.test.ts

```diff
@@ -159,14 +159,14 @@ describe('PreScriptCheck', () => {
       process.env.SKIP_PREFLIGHT_CHECK = 'true';
       await preScriptCheck.default();
 
-      expectCalled(true, false);
+      expectCalled(true);
     });
 
     it('should call all methods and allow react', async () => {
```

Comment:
- removing this test since it is no longer applicable
- will also remove allowUnbundledReact() from env file

Reply:
Can you finish these tasks and then request a review?